crates/cli/src/subcommands/server.rs

```diff
@@ -59,13 +59,6 @@ fn get_subcommands() -> Vec<Command> {
                     .help("The nickname, host name or URL of the server to remove")
                     .required(true),
             )
-            .arg(
-                Arg::new("delete-identities")
-                    .help("Also delete all identities which apply to the server")
-                    .long("delete-identities")
-                    .short('I')
-                    .action(ArgAction::SetTrue),
-            )
             .arg(common_args::yes()),
         Command::new("fingerprint")
```

Comment:
Are we keeping fingerprints?

Reply:
I removed --identity params and IdentityConfig in this PR because they no longer functioned. I know we plan to remove fingerprints, but maybe we should do this in a followup PR since this one is already pretty big?

Reply:
Sure thing.